app/src/main/res/values/strings.xml

```diff
@@ -2,14 +2,14 @@
     <string name="app_name">NervousFish</string>
 
     <string name="loginMessage">Enter your password below</string>
-    <string name="email">Email</string>
+    <!--<string name="email">Email</string>-->
     <string name="password">Password</string>
     <string name="submit">Submit</string>
 
     <string name="title_activity_contact">Contact</string>
-    <string name="msg_email_error">Please enter your email</string>
-    <string name="msg_password_error">Please enter your password</string>
+    <!--<string name="msg_email_error">Please enter your email</string>-->
+    <!--<string name="msg_password_error">Please enter your password</string>-->
     <string name="msg_mismatch">Your password is wrong</string>
     <string name="contact_title">Contact</string>
-    <string name="action_settings">Settings</string>
+    <!--<string name="action_settings">Settings</string>-->
```

Comment:
@ericcornelissen I'm aware that such changes are not preferred in a logging PR, but I had to fix them because otherwise Gradle check would fail each time because of Android Lint. I don't know how the failing code could appear on the master, maybe Travis skips Android Lint, but these issues had to be fixed to let my local check succeed.